README.md

```diff
@@ -7,20 +7,25 @@ So, why not just a jump host? Well, if it's just you and no one else needing acc
 Thinking it could be done simpler, sshmux and sshmuxd got written. It allows you to have a proxy that will *only* permit forwarding to user-specific servers, regardless of method. No other poking around is possible, and no having to allow actual login for anyone to the server running sshmuxd.
 
 # Installation
-sshmuxd can be installed from source (super simple with Go).
-To download source:
+The safest way to install sshmuxd is via github:
 
-	go get github.com/joushou/sshmuxd
+	cd $GOPATH/src/github.com/ && mkdir -p joushou && cd joushou
+	git clone https://github.com/joushou/sshmuxd.git && cd sshmuxd
```

Comment:
Note that I think the "cd to final sshmuxd folder" kinda belongs down in the build/install section.